packages/core/sample-projects/src/type-extends.ts

```diff
@@ -0,0 +1,7 @@
+import '@ckb-js/kuai-core';
+
+declare module '@ckb-js/kuai-core' {
+  export interface KuaiConfig {
```

Comment:
What if extend KuaiConfig in https://github.com/ckb-js/kuai/pull/121/files#diff-e3c4f2238ce558dfb4f70f29fa93d80948da48dafbceba387770fff0176793aeR9 directly

Reply:
The config field type of `KuaiRuntimeEnvironment` is fixed, so if directly extend `kuaiConfig` directly here will be a bit of a problem https://github.com/ckb-js/kuai/pull/121/files#diff- 12405526a43f467c07f021e225857683dcfdfb8088135c5a5ab575a578ac8f49R10-R11

If you think redeclare is not good, I will add a todo first and then change it to generate type

Reply:
> The config field type of `KuaiRuntimeEnvironment` is fixed, so if directly extend `kuaiConfig` directly here will be a bit of a problem [#121 (files)](https://github.com/ckb-js/kuai/pull/121/files#diff-12405526a43f467c07f021e225857683dcfdfb8088135c5a5ab575a578ac8f49R10-R11)
> 
> If you think redeclare is not good, I will add a todo first and then change it to generate type

Marking it `TODO` is fine, let's finish the MVP first